packages/react/ds/src/list/list.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Link } from '../link/link.js';
+import { List } from './list.js';
+
+const meta = {
+  title: 'typography/List',
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'Use lists to make blocks of text easier to read, and to break information into manageable chunks.',
+      },
+    },
+  },
+  argTypes: {
+    items: {
+      control: 'object',
+      type: { name: 'string', required: true },
+      description: 'List of texts to be displayed',
+      defaultValue: ['Item 1', 'Item 2', 'Item 3'],
+    },
+    type: {
+      options: ['normal', 'bullet', 'number'],
```

Comment:
minor: I see you are using enums on html storybook for options but not here, can we please make it consistent? Also, if you can also replace `normal` with enum too or replace `normal` maybe with `none`.

Reply:
good call!